tests/src/test/scala/scalaz/std/SetTest.scala

```diff
@@ -13,6 +13,7 @@ object SetTest extends SpecLite {
   checkAll(isEmpty.laws[Set])
   checkAll(foldable.laws[Set])
   checkAll(FoldableTests.anyAndAllLazy[Set])
+  checkAll(semilattice.laws[Set[Int]])
```

Comment:
We can remove `checkAll(band.laws[Set[Int]])` if `trait Semilattice extends Band` and `semilattice laws` include `band laws`